File: S_3/Programação Orientada a Objetos/Exercicio1.py
chute = float(input("Digite um número que deseja calcular a raiz quadrada:"))
precisao = 0.00000001
b = chute
a = 1
while((b-a) >= precisao):
    b = (a+b)/2
    a = chute/b
print("A raiz é aproximadamente "+str(b))

File: S_3/Programação Orientada a Objetos/Exercicio1.java
import java.io.IOException;
public class Exercicio1 {
    public static void main(String[] args) throws IOException {
        double precisao = (double)0.00000001;
        System.out.println("Digita um valor para calcular a raiz:");
        double num = EntradaTeclado.leDouble();
        double b = num;
        double a = 1;
        while ((b-a) >= precisao){
            System.out.println("Iterando... "+b);
            b = (a+b)/2;
            a = num/b;
        }
        System.out.println("A raiz é aproximadamente: "+ b);
    }
}
